README.md

```diff
@@ -1,393 +1,296 @@
 # Testee
 
-Run your QUnit, Mocha or Jasmine tests from the command line with any browser.
-
-- Runs on all browsers (supporting SocketIO)
-- Many output formats
-- CI integration
-- BrowserStack support
-- Code coverage
-- GruntJS Task
-- Remote URL testing
-
-For questions or discussion checkout our [forums](http://forums.donejs.com/c/testing).
-
-
-## Getting started
-
-You need [NodeJS](http://nodejs.org/) installed in order to use Testee which makes the installation as easy as:
-
-> `npm install -g testee`
-
-`cd` into the main folder of your JavaScript project and run `testee` with your QUnit, Mocha or Jasmine
-test HTML page:
-
-> `testee tests/qunit.html`
-
-
-## Command line testing
-
-The default browser is [PhantomJS](http://phantomjs.org/), just make sure you have it installed anywhere
-on your system.
-
-To run with different local browsers (e.g. Firefox and Safari) use:
-
-> `testee tests/qunit.html --browsers firefox,safari`
-
-Or run multiple tests in multiple browsers:
-
-> `testee tests/unit.html tests/components.html --browsers firefox,safari`
-
-Note that the browser you are using for testing shouldn't be already running (except for PhantomJS which can be started multiple times).
+[![NPM Version][npm-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Dependency Status][david-image]][david-url] [![Gitter Chat][gitter-image]][gitter-url]
```

Comment:
The Greenkeeper PR will add the badge, so that’s not necessary; I would just remove David. I think it‘s easier to _edit_ this file if the badges are on separate lines.